test/modules/referencestrip.js

```diff
@@ -44,4 +44,33 @@
         done();
     });
 
+    // ----------
+    QUnit.test('shadow dom', function(assert) {
+        var done = assert.async();
+
+        class ShadowElem extends HTMLElement {
```

Comment:
Interesting! Is it possible to test the shadow DOM without using ES6? We technically haven't sunset IE8 support (though we're not that diligent about it), so we haven't been introducing modern browser features (except Canvas).

Reply:
I was able to remove the use of `class`, and just attach a shadowRoot to a `div`, this still means that the test will fail in IE11 and below. No other way to test the viewer inside shadowDOM.

do you value the test proving it works in shadowDOM, or having the suite working in IE11 😄 Let me know. i'll make adjustments if you prefer i remove the test. If that is the route though, I think adding a comment in `src/referencestrip.js` explaining why we use `this.element...` should be added so the code isnt accidentally updated in the future